gammapkt.cc

```diff
@@ -226,7 +226,7 @@ void init_xcom_photoion_data() {
         assert_always(Z > 0);
         assert_always(Z <= numb_xcom_elements);
         // convert XCOM data to cgs units already here
-        photoion_data[Z - 1].push_back({.energy = E, .sigma_xcom = sigma * 1e-24});
+        photoion_data[Z - 1].push_back({.energy = E, .sigma_xcom = sigma});
```

Comment:
This a does not match what you have done in your therm_paper, so which one is correct? I suggest you use a suffix (_Mb) if you really must deviate from cgs units to avoid confusing yourself and others.